docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -89,30 +89,39 @@ end group
             group fpc-vscc
                 note over ECC1
                      fpc-vscc is a validation plugin associated with each fpc chaincode.
+                     Any failure below will result in an invalidation of the transaction
                 end note
-                ECC1 -> ECC1 : <args, read/writeset, result> <- extract from transaction
-                ECC1 -> ECC1 : <Enclave_VK, Sig_Enclave, ECC_result> <- extract from result
-
-                create participant ERCC order 30
-                ECC1 -> ERCC++ : queryECCInfo(CC_Id, Hash(Enclave_VK))
-                note right
-                    If FPC enclave is registered, ERCC will return
-                    ECCInfo containing enclave information message;
-                    otherwise ECCInfo is empty.
-                end note
-                return ECCInfo = <ATTESTATION, ...>
-                ECC1 -> ECC1 : check ECCInfo exists
-                note right
-                    If enclave not registered, invalidate transaction
-                end note
-
-                ECC1 -> ECC1 : verify Sig_Enclave with Enclave_VK and <args, read/writeset, ECC_result>
+                alt  MVP
+                  ECC1 -> ECC1 : <args, read/writeset, result> <- extract from transaction
+                  ECC1 -> ECC1 : <Enclave_VK, Sig_Enclave, ECC_result> <- extract from result
+
+                  ECC1 -> ECC1 : verify Sig_Enclave with Enclave_VK and <args, read/writeset, ECC_result>
+
+                  create participant ERCC order 30
+                  ECC1 -> ERCC++ : queryECCInfo(CC_Id, Hash(Enclave_VK))
+                  note right
+                      If FPC enclave is registered, ERCC will return the corresponding (Credentials.Attestated_Data.)CC_Parameters
+                      otherwise CC_Parameters is empty.
```

Comment:
```suggestion
                      If FPC enclave is registered, ERCC will return the corresponding
                      Credentials.Attestated_Data.CC_Parameters;
                      otherwise CC_Parameters is empty.
```